test/Api.IntegrationTest/SecretsManager/Controllers/SecretsControllerTests.cs

```diff
@@ -801,6 +801,70 @@ public async Task GetSecretsByIds_Success(PermissionType permissionType)
         Assert.Equal(secretIds.Count, result!.Data.Count());
     }
 
+    [Fact]
+    public async Task BulkMoveToProjectAsync_Success()
+    {
+        var (org, _) = await _organizationHelper.Initialize(true, true, true);
+        await LoginAsync(_email);
+        var (project, secretIds) = await CreateSecretsAsync(org.Id);
+        var (newProject, _) = await CreateSecretsAsync(org.Id);
+
+        var response = await _client.PostAsJsonAsync($"/organizations/{org.Id}/secrets/move", new BulkMoveToProjectsRequestModel
+        {
+            Secrets = secretIds,
+            Project = newProject.Id,
+        });
+        response.EnsureSuccessStatusCode();
+    }
+
+    [Fact]
+    public async Task BulkMoveToProjectAsync_MoveToUnownedProject_Fails()
+    {
+        var (org, _) = await _organizationHelper.Initialize(true, true, true);
+        var otherOrg = await _organizationHelper.CreateSmOrganizationAsync();
+
+        await LoginAsync(_email);
+        var (project, secretIds) = await CreateSecretsAsync(org.Id);
+        var (newProject, _) = await CreateSecretsAsync(otherOrg.Id);
+
+        var response = await _client.PostAsJsonAsync($"/organizations/{org.Id}/secrets/move", new BulkMoveToProjectsRequestModel
+        {
+            Secrets = secretIds,
+            Project = newProject.Id,
+        });
+
+        Assert.Equal(HttpStatusCode.NotFound, response.StatusCode);
+    }
+
+    [Fact]
+    public async Task BulkMoveToProjectAsync_CannotEditSecret_Fails()
+    {
+        var (org, _) = await _organizationHelper.Initialize(true, true, true);
+
+        var (project, secretIds) = await CreateSecretsAsync(org.Id);
+        var (otherProject, _) = await CreateSecretsAsync(org.Id);
+
+        var (email, orgUser) = await _organizationHelper.CreateNewUser(OrganizationUserType.User, true);
+        await LoginAsync(email);
+
+        var accessPolicies = new List<BaseAccessPolicy>
+        {
+            new UserProjectAccessPolicy
+            {
+                GrantedProjectId = project.Id, OrganizationUserId = orgUser.Id, Read = true, Write = false,
+            },
+        };
+        await _accessPolicyRepository.CreateManyAsync(accessPolicies);
+
+        var response = await _client.PostAsJsonAsync($"/organizations/{org.Id}/secrets/move", new BulkMoveToProjectsRequestModel
+        {
+            Secrets = secretIds,
+            Project = otherProject.Id,
+        });
+
+        Assert.Equal(HttpStatusCode.NotFound, response.StatusCode);
+    }
+
```

Comment:
Nice, thanks for adding integration tests. 

Can we add one for checking SM access? 

Example here https://github.com/bitwarden/server/blob/061253e4285a9320a34189a851c80b788e4b1e09/test/Api.IntegrationTest/SecretsManager/Controllers/SecretsControllerTests.cs#L58C1-L73